internal/services/aws_reservation_service.go

```diff
@@ -48,8 +48,8 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 	reservation.Status = "Created"
 	reservation.Provider = models.ProviderTypeAWS
 	reservation.Steps = 2
-	if payload.Name != nil {
-		newName := config.Application.InstancePrefix + *payload.Name
+	if config.Application.InstancePrefix != "" {
```

Comment:
I belive this is introducing a regression, that the CI is finding.
Now you can't name the instance if the prefix is not configured.

Reply:
Yeah I rushed that in, fixed.